src/client/app/components/about.ts

```diff
@@ -101,9 +101,9 @@ export class About implements AfterViewInit {
             return;
         }
 
-        let changeEnvironmentMessage = this.strings.aboutSwitchEnvironment;
-        changeEnvironmentMessage = changeEnvironmentMessage.replace('{0}', this.configs.find(c => c.value === currentConfigName).name);
-        changeEnvironmentMessage = changeEnvironmentMessage.replace('{1}', this.configs.find(c => c.value === this.selectedConfig).name);
+        let changeEnvironmentMessage = this.strings.aboutSwitchEnvironment.replace(
```

Comment:
Nit: my personal preference is actually to have the line break happen before the ".replace".

    let changeEnvironmentMessage = this.strings.aboutSwitchEnvironment
        .replace('{0}', this.configs.find(c => c.value === currentConfigName).name)
        .replace({1}', this.configs.find(c => c.value === this.selectedConfig).name);

Just you would with a `.then` or `.catch`